src/app/ecosystem-explorer/[slug]/components/Article.tsx

```diff
@@ -16,7 +16,7 @@ type ArticleProps = {
   repo?: string
   twitter?: string
   featuredContent?: string
-  subcategory: string
+  category: string
```

Comment:
Going to add a ticket to look into adding a library for ordering props. 

```Here are the generally accepted best practices for ordering TypeScript props:

```typescript
interface Props {
  // 1. Required props first
  id: string;
  title: string;

  // 2. Optional props next
  subtitle?: string;
  description?: string;

  // 3. Callback/event handlers (usually prefixed with 'on')
  onClick?: (event: React.MouseEvent) => void;
  onSubmit?: (data: FormData) => void;

  // 4. Style-related props last
  className?: string;
  style?: React.CSSProperties;

  // 5. Children prop (if component accepts children)
  children?: React.ReactNode;
}
```

Within each group, props are typically ordered:
- Alphabetically OR
- By importance/frequency of use

Some additional tips:
- Keep related props together (e.g., `min`/`max`, `start`/`end`)
- Consider grouping by domain concepts rather than just required/optional
- Document complex props with JSDoc comments
- Use type literals for props with specific allowed values
```